share/p2p/discovery/discovery.go

```diff
@@ -195,22 +195,31 @@ func (d *Discovery) Advertise(ctx context.Context) {
 }
 
 // discoveryLoop ensures we always have '~peerLimit' connected peers.
-// It starts peer discovery per request and restarts the process until the soft limit reached.
+// It initiates peer discovery upon request and restarts the process until the soft limit is reached.
 func (d *Discovery) discoveryLoop(ctx context.Context) {
 	t := time.NewTicker(discoveryRetryTimeout)
 	defer t.Stop()
+
+	// Add a timer to check every 5 minutes
+	warnTimer := time.NewTicker(5 * time.Minute)
+	defer warnTimer.Stop()
```

Comment:
Also would be nice to make `warnTimer` name consistent with actual type behind it (ticker). Also to make it consistent with out codebase, it would be great to put log interval into constant in the top of the file. It will also allow you to use interval value for log line, instead of using hardcoded "5 minutes"